### ISSUE TITLE
composer.json locked to PHP 7.2

### ISSUE THREAD

**Issue by rhoerr** [EcomDev/magento2-product-preloader#3]
Trying to install with latest Magento causes:

```
Package ecomdev/magento2-product-data-preloader has requirements incompatible with your PHP version, PHP extensions and Composer version:
- ecomdev/magento2-product-data-preloader 1.0.0 requires php ~7.2 which does not match your installed version 8.1.12.
```

Current Magento minimum version is PHP 7.4.

**Comment by IvanChepurnyi** [comment]
Issue is in missing ~8.0 version match, going to add it and run all tests on the version 8.

**Comment by IvanChepurnyi** [comment]
Fixed in 1.1.0
